integration/spark2/src/main/spark2.2/CarbonSessionState.scala

```diff
@@ -280,25 +280,26 @@ class CarbonOptimizer(
   }
 }
 
-class CarbonSqlAstBuilder(conf: SQLConf, parser: CarbonSpark2SqlParser) extends
-  SparkSqlAstBuilder(conf) {
+class CarbonSqlAstBuilder(conf: SQLConf, parser: CarbonSpark2SqlParser, sparkSession: SparkSession)
+  extends SparkSqlAstBuilder(conf) {
 
-  val helper = new CarbonHelperSqlAstBuilder(conf, parser)
+  val helper = new CarbonHelperSqlAstBuilder(conf, parser, sparkSession)
 
   override def visitCreateHiveTable(ctx: CreateHiveTableContext): LogicalPlan = {
     val fileStorage = helper.getFileStorage(ctx.createFileFormat)
 
     if (fileStorage.equalsIgnoreCase("'carbondata'") ||
         fileStorage.equalsIgnoreCase("'org.apache.carbondata.format'")) {
       helper.createCarbonTable(ctx.createTableHeader,
-          ctx.skewSpec,
```

Comment:
@jackylk We should use the right format at the first time, I mean use 2 blanks instead of 4 blanks. We should avoid unrelated modify, so we can keep each MR with less diff. Better to review and better to rebase.